public/style.css

```diff
@@ -21,6 +21,10 @@
   height: 50px;
 }
 
+.form_container {
+  max-width: 330px;
```

Comment:
Yeah guessing this is for website on desktop, we can use tachyons as this has media queries if we want